js/packages/web/src/views/home/index.tsx

```diff
@@ -140,7 +144,7 @@ export const HomeView = () => {
       <Layout>
         <Content style={{ display: 'flex', flexWrap: 'wrap' }}>
           <Col style={{ width: '100%', marginTop: 10 }}>
-            {liveAuctions.length > 1 && (<Row>
+            {liveAuctions.length >= 0 && (<Row>
```

Comment:
May want to adjust this because we purposefully hide this to keep the page clean for single auctions. Maybe do activeKey == LiveAuctionViewState.Resale ? 0 : 1

Reply:
i feel now that we have resales, ended and participated view added, it would be better for users to see these options even if there are no live auctions present